apps/tasks/lib/controllers/my_tasks_controller.dart

```diff
@@ -35,80 +36,26 @@ class MyTasksController extends ChangeNotifier {
       notifyListeners();
     }
 
-    // TODO make grpc request here
-    PatientMinimal patient = PatientMinimal(
-      id: 'patient 1',
-      name: "Victoria Schäfer",
-    );
-    await Future.delayed(const Duration(seconds: 2));
-    _tasks = [
-      TaskWithPatient(
-        id: "task1",
-        name: "Task name - 1",
-        assignee: "",
-        notes: "Some text describing the task",
-        dueDate: DateTime.now().add(const Duration(days: 20)),
-        status: TaskStatus.inProgress,
-        subtasks: [
-          SubTask(id: "subtask 1", name: "Subtask 1"),
-          SubTask(id: "subtask 2", name: "Subtask 2"),
-        ],
-        patient: patient,
-      ),
-      TaskWithPatient(
-        id: "task2",
-        name: "Task name - 2",
-        assignee: "",
-        notes: "Some text describing the task with an very very very long text",
-        dueDate: DateTime.now().add(const Duration(hours: 10)),
-        status: TaskStatus.done,
-        subtasks: [
-          SubTask(id: "subtask 1", name: "Subtask 1", isDone: true),
-          SubTask(id: "subtask 2", name: "Subtask 2"),
-        ],
-        patient: patient,
-      ),
-      TaskWithPatient(
-        id: "task3",
-        name: "Task name - 3",
-        assignee: "",
-        notes: "Some text describing the task",
-        dueDate: DateTime.now().add(const Duration(hours: 1)),
-        status: TaskStatus.todo,
-        subtasks: [
-          SubTask(id: "subtask 1", name: "Subtask 1"),
-          SubTask(id: "subtask 2", name: "Subtask 2", isDone: true),
-        ],
-        patient: patient,
-      ),
-      TaskWithPatient(
-        id: "task4",
-        name: "Task name - 4",
-        assignee: "",
-        notes: "Some text describing the task",
-        dueDate: DateTime.now().subtract(const Duration(days: 20)),
-        status: TaskStatus.inProgress,
-        subtasks: [
-          SubTask(id: "subtask 1", name: "Subtask 1"),
-          SubTask(id: "subtask 2", name: "Subtask 2", isDone: true),
-          SubTask(id: "subtask 3", name: "Subtask 3", isDone: true),
-        ],
-        patient: patient,
-      ),
-      TaskWithPatient(
-        id: "task5",
-        name: "Task name - 5",
-        assignee: "",
-        notes: "Some text describing the task",
-        status: TaskStatus.inProgress,
-        subtasks: [
-          SubTask(id: "subtask 1", name: "Subtask 1"),
-          SubTask(id: "subtask 2", name: "Subtask 2", isDone: true),
-          SubTask(id: "subtask 3", name: "Subtask 3", isDone: true),
-        ],
-        patient: patient,
-      )
-    ];
+    var patients = await PatientService().getPatientList();
+    List<Patient>allPatients = patients.active +
+        patients.discharged + patients.unassigned;
```

Comment:
Could you add this as a getter on the `PatientsByAssignmentStatus` class? Because we will most likely reuse that.

```dart
List<Patient> get all => active + unassigned + discharged;
```